frontend/src/modules/Dashboards/views/DashboardView.js

```diff
@@ -119,7 +119,7 @@ const DashboardView = () => {
   const fetchItem = useCallback(async () => {
     setLoading(true);
     const response = await client.query(getDashboard(params.uri));
-    if (!response.errors) {
+    if (response.data.getDashboard !== null) {
```

Comment:
nit: dispatch all errors if no data

Reply:
In the case of Dashboards if there are errors in getting the dashboard (=no data returned), the errors for the restricted information are not very relevant